core/templates/dev/head/domain/statistics/statistics-domain.constants.ts

```diff
@@ -22,4 +22,6 @@ oppia.constant('LEARNER_ACTION_SCHEMA_LATEST_VERSION', 1);
 
 oppia.constant('FEEDBACK_IMPROVEMENT_CARD_TYPE', 'feedback');
 oppia.constant('PLAYTHROUGH_IMPROVEMENT_CARD_TYPE', 'playthrough');
+oppia.constant('INTERACTION_IDS_WITHOUT_ANSWER_DETAILS', ["EndExploration", "Continue"]);
```

Comment:
I think this value will be defined in the oppia.constant automatically, right?

Reply:
Done